package.json

```diff
@@ -12,35 +12,60 @@
     "develop": "strapi develop",
     "start": "strapi start",
     "strapi": "strapi",
+    "tsc": "tsc --project tsconfig.json --noEmit",
+    "test": "jest --forceExit --detectOpenHandles",
+    "test:with-build": "yarn build && yarn test",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
+    "format": "prettier . --check",
+    "format:fix": "prettier . --write",
+    "check:types": "yarn tsc",
+    "check:lint": "yarn lint",
+    "check:format": "yarn format",
+    "check:build": "yarn build",
+    "check:test": "yarn test",
+    "check:all": "yarn check:types && yarn check:lint && yarn check:format && yarn check:build && yarn check:test",
```

Comment:
This may feel like duplication but having a CI tasks in here (all the `check:` ones) as separate tasks from what you use directly frees us up to make further modifications to these tasks in the future.